extension/js/common/org-rules.ts

```diff
@@ -127,6 +127,10 @@ export class OrgRules {
     return (this.domainRules.flags || []).includes('DEFAULT_REMEMBER_PASS_PHRASE') || this.mustAutogenPassPhraseQuietly();
   }
 
+  public forbidStoringPassPhrase = (): boolean => {
+    return (this.domainRules.flags || []).includes('FORBID_STORING_PASS_PHRASE'); // todo: mustAutogenPassPhraseQuietly() conflict?
```

Comment:
There is indeed a conflict with `mustAutogenPassPhraseQuietly` in the sense that the two together make no sense - generate a pass phrase that gets forgotten soon after. Therefore the two should not be used together, and I'd throw in this method in case the other one is set.

Reply:
I added an error notification to `setup-key-manager-autogen.ts` similar to other notifications

Reply:
good :+1: